packages/react-mutation-mapper/src/component/track/TrackSelector.tsx

```diff
@@ -41,7 +41,7 @@ export default class TrackSelector extends React.Component<
         tracks: [
             TrackName.CancerHotspots,
             TrackName.OncoKB,
-            TrackName.PTM,
+            TrackName.dbPTM,
```

Comment:
Same here, just wanted to make sure that only `dbPTM` is default and `UniprotPTM` is not?